folly/FBString.h

```diff
@@ -2796,7 +2796,7 @@ operator<<(
       os.setstate(_ostream_type::badbit | _ostream_type::failbit);
     }
   }
-#elif defined(_MSC_VER)
+#elif defined(_MSC_VER) || (_CPPLIB_VER <= 800)
```

Comment:
Can we clarify this guard? Does MSVC with latest cpplib have `std::__ostream_insert`?

Reply:
Not that I can see. I tried to compile a reduced sample with VS2017 with latest updates (15.9.7), with C++14 and C++17. It doesn't support this private implementation detail shared by libstdc++ and libc++, and neither does my clang+Dinkumware configuration. I loosened the check to be any version of Dinkumware and a clarifying comment.